src/index.tsx

```diff
@@ -10,7 +10,9 @@ import initSentry from 'config/sentry';
 import {persistor, store} from 'store';
 import 'styles/fonts.css';
 
-initSentry();
+if (process.env.NODE_ENV !== 'development') {
+  initSentry();
+}
```

Comment:
I think we decided not to do this for now.
a PR for this has already been closed